debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/Transaction.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle.logminer.processor;
+
+import java.time.Instant;
+
+import io.debezium.connector.oracle.Scn;
+
+/**
+ * Contract for an Oracle transaction.
+ *
+ * @author Chris Cranford
+ */
+public interface Transaction {
```

Comment:
Let's describe the methods. In particular, what is `started()`? Its name would make you think it returns a `boolean`, but it doesn't.

Reply:
I renamed the method to `start` to avoid the mistaken connotation you referenced as well as documented the methods.

Reply:
Thx!